plugins/woocommerce-admin/client/customize-store/design-without-ai/actions.ts

```diff
@@ -27,8 +27,15 @@ const redirectToIntroWithError = sendParent<
 	DesignWithoutAIStateMachineContext,
 	DesignWithoutAIStateMachineEvents,
 	DesignWithoutAIStateMachineEvents
->( {
-	type: 'NO_AI_FLOW_ERROR',
+>( ( context, event ) => {
+	const errorEvent = event as {
+		type: string;
+		data: { data: { status: number } };
```

Comment:
We can improve the type here:
```suggestion
		data?: { data?: { status: number } };
```